tests/e2e/data/environment.yml

```diff
@@ -0,0 +1,8 @@
+name: app_environment
+channels:
+  - snowflake
+dependencies:
+  - trulens-connectors-snowflake=*
```

Comment:
do we need to create an environment.yml for each test case/suite?

Reply:
No we can reuse the same one if we want. I'm currently just running the single simple notebook and just testing all the imports and printing out some obvious statement.